notebooks/Emukit-constrained-optimization.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<p>A short comment like `expit is scipy's implementation of sigmoid` would be handy</p>

 _Reply via <a href='https://app.reviewnb.com/amzn/emukit/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done</p>